src/index.js

```diff
@@ -90,6 +92,8 @@ function App() {
           <Route exact path="/" component={LandingPage} />
           <Route path="/login" component={LoginPage} />
           <Route path="/register-2" component={HowManyStudents} />
+          <Route path="/instructor-register-1" component={InstructorWelcome} />
```

Comment:
Perhaps we all need to make the index.js routes more organized, put in chronological order or group them up based on the parts of the website they are responsible for... guilty myself!

Reply:
This is a good point, would only help our fellow developers read and understand more smoothly.